.idea/vcs.xml

```diff
@@ -1,36 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!--
-  Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
can you revert changes on this file?

Reply:
I have revert this file in the latest commit. Thank you.